doc/user/content/manage/access-control/_index.md

```diff
@@ -10,116 +10,298 @@ menu:
+Materialize.
+{{< /warning >}}
 
+If you're just getting started and haven't yet decided on an [RBAC strategy](#configuring-advanced-rbac)
+for your organization, we recommend [modifying default privileges](#modifying-default-privileges)
+or — if you just need to move fast and break things — [inviting users as administrators](#inviting-users-as-administrators).
 
-Note that the system catalog uses the abbreviation of the privilege name.
+#### Modifying default privileges
 
-Objects in Materialize have different levels of privileges available to them.
-Materialize supports the following object type privileges:
+Every Materialize region has a `PUBLIC` system role that determines the default
+privileges available to all other roles. On creation, users are automatically
+granted membership in `PUBLIC`, and inherit the privileges assigned to it. By
+default, members of this role (and therefore **all users**) have the following [privileges](/sql/grant-privilege/#privilege):
 
-| Object Type          | Privileges                                |
-|----------------------|-------------------------------------------|
-| `SYSTEM`             | `CREATEROLE`, `CREATEDB`, `CREATECLUSTER` |
-| `DATABASE`           | `USAGE`, `CREATE`                         |
-| `SCHEMA`             | `USAGE`, `CREATE`                         |
-| `TABLE`              | `INSERT`, `SELECT`, `UPDATE`, `DELETE`    |
-| `VIEW`               | `SELECT`                                  |
-| `MATERIALIZED  VIEW` | `SELECT`                                  |
-| `TYPE`               | `USAGE`                                   |
-| `SOURCE`             | `SELECT`                                  |
-| `CONNECTION`         | `USAGE`                                   |
-| `SECRET`             | `USAGE`                                   |
-| `CLUSTER`            | `USAGE`, `CREATE`                         |
+Privilege                            | Scope     |
+-------------------------------------|-----------|
+`USAGE`                            | All types, all system catalog schemas, the `materialize.public` schema, the `materialize` database, and the `default` cluster.|
+`SELECT`                               | All system catalog objects.  |
 
-### Inheritance
+This means that new, non-administrator users have limited access to resources in
+a Materialize region, and don't have the ability to e.g., create new clusters,
+databases, or schemas. To modify the default privileges available to all other
+roles in a Materialize region, you can use the [`ALTER DEFAULT PRIVILEGES`](/sql/alter-default-privileges/)
+command.
 
-Inheritance in RBAC allows you to create roles that inherit privileges from
-other roles. Inheritance only applies to role privileges. Role attributes are
-not inherited. Inheriting privileges allows you to minimize the number of roles you have to manage.
+```sql
+# Use SHOW ROLES to list existing roles in the system, which are 1:1 with invited users
```

Comment:
You may want to call out that a role isn't created until the user first logs on. This has confused many people in the past.